event_reader/src/event_reader.py

```diff
@@ -200,6 +203,7 @@ def run(self):
         if self.consumer:
             for message in self.consumer:
                 self.db.execute(f"INSERT INTO {self.target_table}(event) VALUES ('{message.value}');")
+                # touch to add comments
```

Comment:
I would want to gather an monitor execution time metrics on these queries to make sure they continue to perform in near real time so that they could be used for alerting